doc/src/manual/documentation.md

```diff
@@ -254,10 +254,8 @@ Documentation written in non-toplevel blocks, such as `begin`, `if`, `for`, and
 added to the documentation system as blocks are evaluated. For example:
 
 ```julia
-if VERSION > v"0.5"
-    "..."
-    f(x) = x
-end
```

Comment:
Note correct, but I don't think I can really blame the bot for this. Maybe we can pick a better example in the docs?